apidoc/Titanium/UI/RefreshControl.yml

```
@@ -138,3 +145,13 @@ examples:
 
         $.index.open();
         ```
+---
+name: refreshOffset
```

```suggestion
name: RefreshControlOffset
```

Change in the usage as well